protocol/binary_test.go

```diff
@@ -788,3 +821,137 @@ func TestBinaryEnvelopeSuccessful(t *testing.T) {
 		assert.Equal(t, tt.encoded, buf.Bytes(), "%v: reencoded bytes mismatch")
 	}
 }
+
+func tbinary(v wire.Value) []byte {
+	buf := &bytes.Buffer{}
+	Binary.Encode(v, buf)
+	return buf.Bytes()
+}
+
+func TestReqRes(t *testing.T) {
+
```

Comment:
nit: delete empty line at start of function

I think I read somewhere that upcoming versions of golint will complain about this